src/reference/asciidoc/whats-new.adoc

```diff
@@ -1,3 +1,11 @@
+=== What's new in 2.0
+
+==== Support for Kafka timestamps
+
+`KafkaTemplate` now supports API's to add records with timestamps. The behavior of the recorded timestamp is dependent on the
```

Comment:
There is a recommendation for makrdowns to right each sentence on its own line: http://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line.

Not critical right now, but for the future contribution.

Plus we typically make a short teaser sentences in the "What's new" with the link to an appropriate chapter with full description of the feature.